sway/commands/move.c

```diff
@@ -859,13 +848,18 @@ static struct cmd_results *cmd_move_to_position(int argc, char **argv) {
 		}
 		container_floating_move_to(container, lx, ly);
 		return cmd_results_new(CMD_SUCCESS, NULL);
+    }
+
+	if (!argc) {
```

Comment:
You are now checking if there are any arguments by inspecting `argc`, *after* said arguments are used by accessing `argv` (line 829, 832, 835). This is not correct.